agreement/pom.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.fiware.tmforum</groupId>
+    <artifactId>agreement</artifactId>
+    <version>0.1</version>
+    <parent>
+        <groupId>org.fiware</groupId>
+        <artifactId>tmforum</artifactId>
+        <version>0.1</version>
+    </parent>
+
+    <properties>
+        <tmforum.api.url>https://tmf-open-api-table-documents.s3.eu-west-1.amazonaws.com/OpenApiTable/4.0.0/swagger/TMF651-Agreement-v4.0.0.swagger.json</tmforum.api.url>
+        <module.ctk.url>https://tmf-open-api-table-documents.s3.eu-west-1.amazonaws.com/OpenApiTable/4.0.0/ctk/TMF651-Agreement-v4.0.0.zip</module.ctk.url>
+        <module.ctk.script-folder>TMF651-Agreement</module.ctk.script-folder>
+        <module.ctk.run-script>Mac-Linux-RUNCTK.sh</module.ctk.run-script>
+        <module.ctk.base-path>/tmf-api/agreementManagement/v4/</module.ctk.base-path>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.fiware.tmforum</groupId>
+            <artifactId>common</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.fiware.tmforum</groupId>
+            <artifactId>mapping</artifactId>
```

Comment:
Hi, I'm sorry, but your basically one day to late:) This dependency(even thought its still the same code), was extracted into a library. Please update to this.

Reply:
Good morning, I have updated the mapping libraries from the agreement module